src/Compilers/CSharp/Portable/Symbols/AssemblySymbol.cs

```diff
@@ -494,6 +496,21 @@ internal bool RuntimeSupportsByRefLikeGenerics
             }
         }
 
+#nullable enable
+        private bool _overrideRuntimeSupportsAsync;
```

Comment:
Consider adding a PROTOTYPE marker to track that this should be removed when we have a runtime with support